packages/e2e-tests/src/hooks/scenarioTagRunner.ts

```
@@ -5,15 +5,25 @@ import consoleManager from '../utils/consoleManager';
 
 import allure from '@wdio/allure-reporter';
 import testContext from '../utils/testContext';
+import PidMonitor from '../support/PidMonitor';
+
+const monitor = PidMonitor.getInstance();
 
 // eslint-disable-next-line no-unused-vars
 Before(async () => {
   if (String(process.env.SERVICE_WORKER_LOGS) === 'true') {
     await consoleManager.startLogsCollection();
   }
+  const pidMonitorInitialized = await monitor.init();
+  if (!pidMonitorInitialized) return;
+  monitor.start();
```

Maybe it's better to try / catch start()? as it may throw exception when pid is undefined, what means initialization failed?